app/build.gradle

```diff
@@ -6,8 +6,8 @@ android {
         applicationId "com.allybros.superego"
         minSdkVersion 26
         targetSdkVersion 33
-        versionCode 20
-        versionName "2.0.0"
+        versionCode 21
+        versionName "2.0.1"
```

Comment:
should it be 2.1.0, not be ?

Reply:
Depends on preference, we can discuss if it is a minor version or patch

Reply:
Okay we will discuss. But now it remains as is